plugins/modules/python_distribution.py

```diff
@@ -37,11 +37,17 @@
       - "Warning: This feature is not yet supported."
     type: str
     required: false
+  remote:
+    description:
+      - Name of the remote source to add to this publication
```

Comment:
```suggestion
      - Name of the remote source to add to this distribution
```
Could also say "index".

Reply:
Fixed.